util/cmake/SeqAnSimdUtility.cmake

```diff
@@ -185,18 +185,22 @@ set(SEQAN_SIMD_AVX512_KNL_SOURCE
 "#include <cstdint>
 #include <immintrin.h>
 #include <iostream>
+#include <random>
 
 int main() {
+  std::random_device r;
```

Comment:
I strongly advice against using random stuff in tests and build stuff. 
Why does it need to be randomly generated?

Reply:
Uhm, this produces a binary that tests if some errors occur when using sse4/avx2/avx512. If I don't include some randomness, the compiler will optimize everything away and tests nothing.

Reply:
use the volatile keyword to prevent a variable from being optimised away!

Reply:
did not work

Reply:
Ok I understand the problem now. 
Let's keep it this way